app/components/list/index.tsx

```diff
@@ -25,7 +26,7 @@ export const List = ({
   headerChildren?: ReactNode;
   hideFirstHeaderColumn?: boolean;
   /** Function to be passed if the rows of the table should navigate */
-  navigate?: (id: string) => void;
+  navigate?: (id: string, item: ListItemData) => void;
```

Comment:
I am not sure why we need that. Can you explain please

Reply:
Discussed on discord